src/config/inversion-fixes.config

```diff
@@ -1317,6 +1317,13 @@ CSS
 
 ================================
 
+mediawiki.org
+
+IGNORE IMAGE ANALYSIS
```

Comment:
IGNORE IMAGE ANALYSIS is only available for the [dynamic-theme-fixes.config](https://github.com/darkreader/darkreader/blame/master/src/config/dynamic-theme-fixes.config) Did you meant to place your fix their?

Reply:
Yup, I will place that there. Thank you!